app/components/footer.js

```diff
@@ -22,9 +100,40 @@ export default function Footer() {
     const classes = useStyles()
     return (
         <div className={classes.footer}>
-            <a href='https://enciv.org/terms' target='_blank' rel='noreferrer'>
-                Terms
-            </a>
+            <div className={classes.linksContainer}>
+                <div className={classes.footerUnpoll}>
+                    <a href='#' target='_blank' className={classes.aLink}>
+                        Unpoll
+                        <img src={ExternalLink} alt='ExternalLink' style={{ marginLeft: '1%' }} />
+                    </a>
+
+                    <p className={classes.pText}>Vestibulum massa id donec gravida purus in tellus aliquam arcu?</p>
+                </div>
+                <div className={classes.footerEnCiv}>
+                    <a href='https://enciv.org/' target='_blank' className={classes.aLink}>
+                        EnCiv
+                        <img src={ExternalLink} alt='ExternalLink' style={{ marginLeft: '1%' }} />
+                    </a>
+                    <p className={classes.pText}>Vestibulum massa id donec gravida purus in tellus aliquam arcu?</p>
+                </div>
+            </div>
+            <div className={classes.donateContainer}>
+                <div>
+                    <p className={classes.pDonate}>amet elementum euismod molestie facilisis varius</p>
+                    <button className={classes.btn}> Donate</button>
```

Comment:
When someone clicks on the donate button, send them to this page.  You can do something like onClick={e=>location.href="https://www.paypal.com/donate/?cmd=_s-xclick&hosted_button_id=H7XBVF5U2C9NJ&source=url"} to go to another page from a button. But test this, it might not be completely right.